src/components/SignIn/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import validator from 'validator'
+import Input from '../Input'
+
+const StyledForm = styled.form`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid teal;
+    margin: 10px;
+    padding: 25px;
+    border-radius: 15px;
+`
+const ErrorMessage = styled.p`
+    margin-top: 15px;
+    color: red;
+    width: 20rem;
+`
+
+const StyledButton = styled.button`
+    border-radius: 10px;
+    padding: 10px;
+    border: 2px solid teal;
+    ${(props) =>
+        props.disabled &&
+        `
+        border-color: red;
+        cursor: not-allowed;
+  `}
+`
+
+const SignIn = () => {
+    const [userName, setUserName] = useState('')
+    const [password, setPassword] = useState('')
+    const [errorExist, setErrorExist] = useState(false)
+    const [errorMessage, setErrorMessage] = useState('')
```

Comment:
Is it possible not to store this values here, but check it inside `Input` component?